### ISSUE TITLE
Ask about how to calculate the ’accuracy‘ in the paper

### ISSUE THREAD

**Issue by chr-523** [Yu-Utah/Gamora#8]
​	      The code's function ’write_txt‘ uses pre-trained or only trained models to predict the root and shared labels of the input design. However, in this article, accuracy refers to the precision of extracting the full-adder and half-adder. This article may not have clearly expressed how to extract adders after obtaining labels and accurately calculate  the "accuracy". 

​	      In my understanding, it may be using ABC tools for extraction, but I don't know how to start. I want to know how to extract full-adder and half-adders from roots and shared labels, and how to calculate accuracy. I really need help to remove my confusion. thank you.
